### PR TITLE
Add warning message if the bulk download speed falls below a certain …

### LINKED ISSUE #256: Document download limitation from marktstammdatenregister.de
I have experienced now for a few times that the server from marktstammdatenregister.de sets a download speed limitation. At those times the bulk download works only with ~10kB/sec and needs several hours. Should we monitor the speed and give out a warning to the user if the speed is too slow.

"Warning: Currently the servers from marktstammdatenregister have restricted the download speed. Please try it again another time:"

### PR DESCRIPTION
**Pull request by deniztepe**
…limit #256

* The limit is set to 100kB/s
* The unit of tqdm bar is set to MB, since each iteration corresponds to one megabyte

closes #256

### CODE REVIEW

**Comment by deniztepe** [comment]
The warning message should work fine now. Testing it is not intuitive, because you have to limit the download speed manually. Instead you can set the warning limit higher to something like 12 MB/s in `open_mastr/xml_download/utils_download_bulk.py:58` and run the following script:

```
from open_mastr.xml_download.utils_download_bulk import download_xml_Mastr
import os

# xml download
save_path = os.path.join(
            os.path.expanduser("~"), ".open-MaStR", "data", "Gesamtexport_test"
        )
download_xml_Mastr(save_path=save_path)
```

File: open_mastr/xml_download/utils_download_bulk.py [hunk 1]
```diff
@@ -35,13 +35,17 @@ def download_xml_Mastr(save_path: str) -> None:
         "Download has started, this can take several minutes."
         "The download bar is only a rough estimate."
     )
+    warning_message = (
+        "Warning: The servers from MaStR restricts the download speed."
```
Comment (FlorianK13) [inline open_mastr/xml_download/utils_download_bulk.py]: servers restrict (no s)

**Review by FlorianK13 — CHANGES_REQUESTED** [review]
Only a small spelling mistake.

File: open_mastr/xml_download/utils_download_bulk.py (hunk 1 above)
Comment (deniztepe) [reply open_mastr/xml_download/utils_download_bulk.py]: done

**Review by FlorianK13 — APPROVED** [review]
Looks good.

**Pull request merged**